modules/rhcos-install-iscsi-manual.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-manual_{context}"]
+= Installing {op-system} manually on an iSCSI boot device
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target that you want to install {op-system} on.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
+    --login
+----
+<1> The IP address of the target portal.
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments:
++
+[source,text]
+----
+$ coreos-installer install \
+    /dev/disk/by-path/ip-<IP_address>:<port>-iscsi-<target_iqn>-lun-<lun> \
+    --append-karg rd.iscsi.initiator=<initiator_iqn> \
+    --append.karg netroot=<target_iqn> \
+    --console ttyS0,115200n8
+    --ignition-file <path_to_file>
+----
+
+. Unmount the iSCSI disk:
++
+[source,text]
+----
+$ iscsiadm --mode node --logoutall=all
+----
+
+This procedure can also be performed using the `coreos-installer iso customize` or `coreos-installer pxe customize` subcommands.
```

Comment:
I wouldn't mention these if we don't show them. Do they warrant an additional procedure or Optional: steps?